core-bundle/src/Command/MigrateCommand.php

```diff
@@ -537,4 +546,40 @@ private function compileTableWarnings(Table $table): array
 
         return $warnings;
     }
+
+    private function validateDatabaseVersion(bool $asJson): bool
+    {
+        // TODO: Find a replacement for getWrappedConnection() once doctrine/dbal 4.0 is released
+        $driverConnection = $this->connection->getWrappedConnection();
+        $currentPlatform = \get_class($this->connection->getDatabasePlatform());
```

Comment:
<img width="903" alt="" src="https://user-images.githubusercontent.com/1192057/184342447-65c15db3-485c-46de-b931-bd28b72ffbcf.png">

Is this going to be a problem?

Reply:
I don’t think so, see https://github.com/contao/contao/pull/5133

Reply:
See https://github.com/ausi/contao-contao/pull/4 😄

Reply:
Merged.